projects/js-packages/components/components/record-meter-bar/index.tsx

```diff
@@ -29,6 +29,10 @@ export type RecordMeterBarProps = {
 	 * The items to display in Record meter.
 	 */
 	items: Array< RecordMeterBarItem >;
+	/**
+	 * The formatting style for legend item display. If not provided, it defaults to showing legend label after count
+	 */
+	showLegendLabelBeforeCount: boolean;
```

Comment:
This should have been marked as optional for backward compatibility. It [breaks TS types](https://github.com/Automattic/jetpack/runs/6604487218?check_suite_focus=true) for existing consumer components.

@anomiex another reason to ship #24329

Reply:
Sorry we both missed this in review @manzoorwanijk!

Would you like us to spin up a PR to address this specifically or are you happy to cover it in #24329?